frontend/packages/dev-console/integration-tests/features/addFlow/create-from-templates.feature

```diff
@@ -1,54 +1,36 @@
 @add-flow
 Feature: Create Application using Templates
-              As a user, I want to deploy the application using Templates provided in Developer Catalog
+              As a user, I want to deploy the application using different types Templates provided in Developer Catalog
 
         Background:
             Given user is at developer perspective
-              And user is at Developer Catalog page
               And user has created or selected namespace "aut-addflow-templates"
+              And user is at Developer Catalog page
               And user is at Templates page
 
 
-        Scenario: Deploy CI/CD Application using Catalog Templates
-             When user selects CI/CD type
-              And user searches and selects Template card "Jenkins" from catalog page
-              And user clicks on Instantiate Template button
-              And user clicks on Create button
+        @smoke
+        Scenario: Deploy Application using CI/CD type of Catalog Templates
+             When user selects "CI/CD" from Templates type
+              And user searches and selects Template card Jenkins from catalog page
+              And user clicks Instantiate Template button on side bar
+              And user clicks create button
```

Comment:
We can more this scenarios to create-from-catalog.feature

Reply:
Updated